examples/two-columns-layout/src/app.tsx

```diff
@@ -98,8 +118,25 @@ autocomplete({
       </div>
     );
 
+    const faqPreviewContext = previewContext as FaqHit;
+    const isFaqPreview = Boolean(faqPreviewContext?.title);
+    const faqSection = (
+      <FaqPreview components={components} {...faqPreviewContext} />
+    );
+
+    const onMouseLeave = () => {
+      const el = document.querySelector('[data-active=true]');
+      el?.removeAttribute('data-active');
```

Comment:
You should be able to do this with `setActiveItemId`, manual DOM updates shouldn't happen.

Reply:
See my comment above https://github.com/algolia/autocomplete/pull/913#discussion_r823824012